packages/scripts/src/commands/app-data/convert/processors/xlsxWorkbook.ts

```diff
@@ -61,6 +61,18 @@ export class XLSXWorkbookProcessor extends BaseProcessor<IContentsEntry> {
           worksheet[cellId].v = html;
         }
       });
+      // If authored value was a percentage, override converted decimal value to preserve percentage representation
+      Object.keys(worksheet).forEach((cellId) => {
```

Comment:
nit(blocking): The codeblock above already loops over all the cells so would probably be a little bit tidier to include the logic there instead of looping over for a second time.